### ISSUE TITLE
[epic] Reboot ecosystem site and use it for Cohere data

### ISSUE THREAD

**Issue by matthewmccarthy11** [life-itself/second-renaissance#92]
Based on our analysis (see notes below) we plan to reboot this ecosystem repo and have ecoystem.lifeitself.org **and** integrate the cohere organizations into here (b/c overlap in content and purpose).

## Acceptance 

- [x] Have a single “wiki” including at least current LI ecosystem, Cohere+, ORA
  - [x] #101 
  - [x] #143
  - [ ] Contributor guide (updated) - document process for contributing
  - [x] #139 **🚧2023-11-28 we have added about 20. Need to check quality.**
  - [ ] #145
- [x] Have “sub-landing pages” for specific initiatives
  - [ ] Landing page for cohere ...
  - [ ] Landing page for ORA
- [ ] (?) Have tags or other way of filtering by different projects (? what's the underlying need?)
- [x] Implement this via a markdown-based wiki edited in obsidian, synced via gdrive and/or github, and published via github + flowershow
- [ ] #144 

## 2023-11-29

- [x] create template for 2021 metadata (quick and dirty) **✅ and tracked in https://github.com/life-itself/ecosystem/issues/143**
- [x] open new issue to implement plan close current issue and mention the new one **Old: #143. New: #150** 
- [x] identify 3-5 orgs we would like to have good profiles of and assess their current quality **4 orgs identified in this doc: https://docs.google.com/document/d/1byQxCIhsJXornGwe2gnaMVMt4x0YBVvIdcE8ufk8dD4/edit#heading=h.3o3oh5ilul5o**
  - [x] identify
  - [x] paste in current content and assess **I did a quick review of 1 org (The Laszlo Institute) and put a few comments and questions in the above doc**
- [ ] #152

### Set-up

- [x] Independent ecosystem repo (and website) back (Why? Because can create, edit build independent of main site. Currently: Have ecosystem list on https://github.com/life-itself/community/tree/main/content/ecosystem and at https://github.com/life-itself/ecosystem)
  - [x] Resurrect https://github.com/life-itself/ecosystem
  - [x] Check nothing added in new location since it moved - see https://github.com/life-itself/community/commit/594422d31ed33c11698ff1d0c01ff24b613aee82 and compare with old site @matthewmccarthy11 **so far haven't seen any significant website changes**
  - [x] Remove redirect and resurrect [ecosystem.lifeitself.org](http://ecosystem.lifeitself.org/) **what do we mean by remove redirect?** @rufuspollock **✅2023-11-23 removed old redirect and added new one from lifeitself.org/ecosystem => ecosystem.lifeitself.org** 
- [x] 👩‍💻 Refactor ecosystem site to use latest flowershow @rufuspollock #101  **✅2023-11-16**
- [x] Prep work to create process for adding/preparing profiles e.g.
  - [x] A “stub” template - an outline of the data we want for each org **CT updated the [template](https://github.com/life-itself/deliberately-developmental-spaces/blob/main/templates/organization.md) in DDS repo to create a stub template for DDS project data**
  - [x] Stub templates for other project data?
  - [x] Create template on Obsidian
### Situation

- Existing ecosystem mapping site which has c. 100 (?) organisation profiles on it
- Did a mapping of ‘polycrisis response’ orgs
- Cohere+ project will have a database of 300+ orgs with significant overlap with previous ecosystem mappings 
- A deliverable of the Cohere+ project is an ‘interactive digital resource’, i.e. some kind of website which presents our map
  - A key component of the cohere+ digital map is that it is ‘interactive’

### Complication

- Matthew (and Catherine?) is/are not proficient in ‘website building’, and would need guidance in this area
 - Either for publishing on LI ecosystem or other site
- There is quite some overlap between the orgs we have been mapping for various projects but the kinds of data we have collected on orgs is different for each project (TODO: detailed needed)

### Questions

- What is the end goal (or foreseeable goal) for 'ecosystem mapping’ in LI? **✅2023-11-28 we have two distinct mapping efforts. See https://github.com/orgs/life-itself/discussions/924**
- How can we present the data that we have collected as part of various mapping projects in a way that is useful and valuable to us and to others? **✅2023-11-28 post it up online in the simplest way possible initially with index pages to provide an introduction**
- How can the information we have gathered and curated in various mapping projects be of maximum use to others (in a resource efficient way)? **✅2023-11-28 see previous point**
  - And in a way that is 'engaging'? **✅2023-11-28 do work to analyse the collected material and provide "ways in" that address questions people may have**

### Hypothesis

- Have a single “wiki” including at least current LI ecosystem, Cohere+, ORA because:
  - Some overlap between entities
  - Shared conceptual knowledge (that can also go in wiki part)
  - Less work than separate
- Have “sub-landing pages” for specific initiatives
- Have tags or other way of filtering by different projects
- Implement this via a markdown-based wiki edited in obsidian, synced via gdrive and/or github, and published via github + flowershow

## Background brainstorm

- Directories alone are not that useful (although they can be to us, especially when they help us to answer specific questions)
- People don’t generally say "wow that list or orgs was so helpful" but tend to say "wow that visualisation was really interesting"
- Need to be able to pull out groups of orgs from directory, e.g. Cohere+ ones DDS ones
- Don't want landing page to a whole long list of orgs – more likely short intros to the different projects / groups of orgs 
- Don't know how many external contributions we are really gonna get 
- Directories in themselves are not necessarily all that helpful (depends)
- Individual profiles getting more rich makes it more helpful 
- Obsidian wiki model can be a good idea as it allows for integration of orgs and concepts 
- There are specific needs we have as it relates to the deliverables in each project 
- The mixture of things- orgs and concepts- makes it nice
- For an all in one wiki, it would be possible to have overlap- allows for surprise/ journeys 
- Its okay to be opinionated and 'sensitive' to how we feel about it- using our best judgement to understand the org 
- Difficult to get a sense for how many 'contributions' there can be
- Part of the value is in the investigation and our own feelings about it
- One option: private repo and public website 
- How might we leverage our own spot in the 'overall ecosystem'as it relates to cohere+ and dds

**Comment by catherinet1** [comment]
> @catherinet1 if you wouldn't mind checking this and providing feedback whenever you get the chance

seems good to me, thanks @matthewmccarthy11

**Comment by catherinet1** [comment]
hi @rufuspollock -- looks as though @matthewmccarthy11 has done a check for changes between old site and new location. can you do the 'remove redirect and resurrect ecosystem site' and 'refactor to latest flowershow tasks'?

**Comment by catherinet1** [comment]
cc @BoazBF -- btw, this is the issue re integrating mapping projects data into one wiki, starting with DDS data

**Comment by catherinet1** [comment]
I created/updated the organisation [template](https://github.com/life-itself/deliberately-developmental-spaces/blob/main/templates/organization.md) in the DDS repo to use as a stub template here. (did not create template in this ecosystem repo as not sure whether can add things to repo before redirect/resurrection happens). 

@rufuspollock does that work as a stub template? i assume we will have different templates for the data from different mapping projects and that that is okay?

**Comment by rufuspollock** [comment]
@matthewmccarthy11 can we get the rest of the context / notes from our call copied into the issues (maybe in a notes section) - that was important context. The description of the issue could also be clarified / made more succinct. 🙏

**Comment by rufuspollock** [comment]
> I created/updated the organisation [template](https://github.com/life-itself/deliberately-developmental-spaces/blob/main/templates/organization.md) in the DDS repo to use as a stub template here. (did not create template in this ecosystem repo as not sure whether can add things to repo before redirect/resurrection happens).

Looks good. I think you could have some more commentary about what the sections are for - cf https://github.com/life-itself/community/blob/main/content/templates/projects.md (the comments / or default values). Maybe also a bit nicer spacings, 2nd level headings. However, this is definitely good to go for now.

**Comment by rufuspollock** [comment]
> hi @rufuspollock -- looks as though @matthewmccarthy11 has done a check for changes between old site and new location. can you do the 'remove redirect and resurrect ecosystem site' and 'refactor to latest flowershow tasks'?

I will look at this - suggest we have a meeting today so you are unblocked.

A redirect is the fact that if you visit ecosystem.lifeitself.org right now you get redirected ...

**Comment by rufuspollock** [comment]
@matthewmccarthy11 as mentioned above getting the original version of the SCQA and overview would be really helpful. Currently when re-reading this i'm not clear what this is really about.

**Comment by matthewmccarthy11** [comment]
Hi @rufuspollock put in the notes and fixed the SCQA- apologies for not doing that sooner. Seems like based on the DDS conversation we aren't entirely set on this exact approach atm. Just so we're clear on how we want the three interact (LI, DDS, and cohere)- would it be like the picture attached below? As in, DDS and cohere both have a spot in the LI ecosystem (perhaps dds later on) but they also have their own 'spot' in the sense that the deliverable for cohere will stand on its own and the DDS site will eventually stand on its own

![li-ecosystem](https://github.com/life-itself/ecosystem/assets/131390186/c88192d7-a1d9-447f-a043-08f4c2321039)

**Comment by rufuspollock** [comment]
@matthewmccarthy11 i'm still pretty confused and it's not very succinct 😉 can you please try to make your communication a bit more succinct and clear 😄 (including even your last comment where when parsing it i struggle to undersand - quickly - what you are asking / suggesting).

Re the SCQA: it's massive and undistilled. I now struggle to recollect who wrote that SCQA and when. At the very least we want to pull up the conclusion from it to the start of the issue and move the bulk of it down the description to an appendix  so we get acceptance and tasks nearer the top.

Also the title of this issue should be clearer.

Overall the biggest feedback is: work to communicate more clearly (i recommend reading minto's the pyramid principle which is in our library).

**Comment by matthewmccarthy11** [comment]
In response to the whatsapp message @rufuspollock ... To get a screenshot of the new orgs from cohere+ that have been published in the ecosystem, we need to know the location of these web pages. Although they build successfully through GitHub, it is not easy to find them on the ecosystem site, and seemingly do not appear on the site. None of the headings available (about, contribute, glossary, report, contact, a project of LI and collaborators) provides a way to access these new orgs. All of the hyperlinks as well do not provide access to the new orgs.

Have also tried scrolling through the logs on cloud fare to find a url of the organizations but was unable to find one. 

Hypothesis: Create a new heading on the ecosystem website which allows us to access these orgs.

**Comment by rufuspollock** [comment]
> In response to the whatsapp message @rufuspollock ... To get a screenshot of the new orgs from cohere+ that have been published in the ecosystem, we need to know the location of these web pages. Although they build successfully through GitHub, it is not easy to find them on the ecosystem site, and seemingly do not appear on the site. None of the headings available (about, contribute, glossary, report, contact, a project of LI and collaborators) provides a way to access these new orgs. All of the hyperlinks as well do not provide access to the new orgs.
> 
> Have also tried scrolling through the logs on cloud fare to find a url of the organizations but was unable to find one.
> 
> Hypothesis: Create a new heading on the ecosystem website which allows us to access these orgs.

Hi, can we break this out to a separate issue - i don't think it is really part of this. there are also a couple of different points in here - and i think the last point about creating a landing page for cohere and something in navbar is a good idea ...

**Comment by matthewmccarthy11** [comment]
I've just made an issue #139 to outline the first steps in how we might have them appear on the site. 

Hypothesis: based on the way the site builds, the organizations in the 'tbd' folder don't appear on the website, it is only orgs in the 'profiles folder', which appear.
